### ISSUE TITLE
Not working on Firefox 57

### ISSUE THREAD

**Issue by quinze26** [benbasson/newtabhomepage#30]
This add-on hasn't worked for me since Firefox 57. When a new tab is created, only a blank page appears although the address bar reads 'Extension (New tab homepage)'. I've had several subsequent updates of Firefox but still the add-on doesn't work.

**Comment by Mishkatel** [comment]
Not working for me. Firefpx 57 on Windows 7.

**Comment by quinze26** [comment]
It would be nice to have some feedback from the developer on this. Is the problem likely to be fixed?

**Comment by tkburroughs** [comment]
The problem may be that NTHP does not work with a local file, so syntax like "file:///c:/users/...."
I recently moved up to the latest Firefox ESR (60.2.0.esr), and NTHP stopped working. Symptoms as described above.  I found that on another system I also moved up and it worked fine..... the difference being my homepage started with 'https://'.  On my non-working system, if I switch my homepage to something with 'https://', tada, NTHP started working.  Switch back to my local file, and again stopped working.
